app/config/Properties.scala

```diff
@@ -0,0 +1,22 @@
+package config
+
+import com.typesafe.config.{ Config, ConfigFactory }
+
+import scala.concurrent.duration._
+
+/**
+ * Created by coolit on 23/11/2017.
+ */
+object Properties {
+
+  private val config: Config = SBRPropertiesConfiguration.envConfig(ConfigFactory.load())
```

Comment:
As discussed, this isn't good practice since we should inject the `Config` / `Configuration` instances.  However, this will be improved in my PR so we can wait for that.